cypress/e2e/specialFeatures/liteSiteWeight/helpers/getPageSizeInBytes.js

```diff
@@ -0,0 +1,6 @@
+export default request => {
+  const getSize = `echo ${JSON.stringify(request)} | gzip | wc -c`;
```

Comment:
I am not sure if it will make a huge difference performance wise, but we could do a check that request is not empty before running the exec command?

I am seeing quite a few: 
```
exec echo {} | gzip | wc -c
```

in the cypress output which is not really required?

We could do this:
```
if (request) {
  return getSize
}
return 0
```